client/src/index.jsx

```diff
@@ -1,53 +1,51 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import $ from 'jquery';
 import axios from 'axios';
-import Description from './components/description.jsx';
-import Photos from './components/photos.jsx';
+import Description from './components/description';
+import Photos from './components/photos';
```

Comment:
Interesting.  I got the Airbnb error to remove the .jsx extension too but when I removed it, it no longer was able to load the file.  I'll go back and try again.

Reply:
Yeah that threw me for a bit of a loop. I think what ended up fixing it was adding:
 resolve: {  extensions: ['.js', '.jsx'],  } in the webpack.config, but honestly, I think i tried like 10 different things to make it work so it mightve been something else.